hex/spec/fixtures/projects/private_repo/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM hexpm/elixir:1.13.3-erlang-24.2.1-alpine-3.15.0 as build
```

Comment:
This file should not live in `fixtures`, we can have it under `hex/spec/Dockerfile.private_hex_server` instead